mev-relay-rs/src/relay.rs

```diff
@@ -104,24 +124,24 @@ pub struct Inner {
 #[derive(Debug, Default)]
 struct State {
     execution_payloads: HashMap<BidRequest, ExecutionPayload>,
+    current_slot: Slot,
 }
 
 impl Relay {
-    pub fn new(
-        genesis_validators_root: Root,
-        beacon_node: Client,
-        secret_key: SecretKey,
-        context: Arc<Context>,
-    ) -> Self {
+    pub fn new(beacon_node: Client, genesis_validators_root: Root, context: Arc<Context>) -> Self {
+        let key_bytes = [1u8; 32];
+        let secret_key = SecretKey::try_from(key_bytes.as_slice()).unwrap();
         let public_key = secret_key.public_key();
         let validator_registry = ValidatorRegistry::new(beacon_node);
+
         let inner = Inner {
             secret_key,
             public_key,
             genesis_validators_root,
             builder: NullBuilder,
             validator_registry,
             context,
+            //TODO: Can we rely on the future to poll and update this upon initialization instead of initializing it?
```

Comment:
no

it just allocates the default state here, I don't see a reason to move this somewhere else